assets/js/base/components/reviews/types.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { EmptyObjectType } from '@woocommerce/types';
+
+export type review =
+	| {
+			date_created: string;
+			date_created_gmt: string;
+			formatted_date_created: string;
+			product_name: string;
+			product_permalink: string;
+			review: string;
+			reviewer: string;
+			id: number;
+			product_id: number;
+			product_image: {
+				alt: string;
+				thumbnail: string;
+				name: string;
+				sizes: string;
+				src: string;
+				srcset: string;
+			};
+			reviewer_avatar_urls: { [ size: string ]: string };
+			verified: boolean;
+			rating: number;
+	  }
+	| EmptyObjectType;
```

Comment:
> Not sure about | EmptyObjectType. Is there a particular reason why did you add?

We have the possibility of an empty object [here](https://github.com/woocommerce/woocommerce-blocks/blob/trunk/assets/js/base/components/reviews/review-list-item/index.js#L153). If I remove it, it will throw TypeScript errors. I'm happy to learn if there is any other way we can approach this issue. 🙂